.github/workflows/release-init.yml

```
@@ -0,0 +1,19 @@
+name: '[Release] Init release (create release branch for code freeze)'
+
+on:
+  workflow_dispatch:
+   
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)